app/views/idv/phone_errors/_warning.html.erb

```diff
@@ -19,6 +19,17 @@ locals:
           text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
           new_tab: true,
         },
+        {
+          url: help_center_redirect_url(
+            category: 'verify-your-identity',
+            article: 'phone-number',
+            flow: :idv,
+            step: :phone,
+          ),
+          text: t('idv.troubleshooting.options.learn_more_verify_by_phone'),
+          new_tab: true,
+          location: local_assigns.fetch(:name, 'warning'),
```

Comment:
I missed previously, but this also is in the troubleshooting option, not the URL options, so would not be tracked as expected.

Reply:
see https://github.com/18F/identity-idp/pull/8115